sentry/tests/test_client.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2016-2017 Versada <https://versada.eu/>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+import sys
+from unittest.mock import patch
+
+from sentry_sdk.integrations.logging import _IGNORED_LOGGERS
+from sentry_sdk.transport import HttpTransport
+
+from odoo import exceptions
+from odoo.tests import TransactionCase
+from odoo.tools import config
+
+from ..const import to_int_if_defined
+from ..hooks import initialize_sentry
+
+GIT_SHA = "d670460b4b4aece5915caf5c68d12f560a9fe3e4"
+RELEASE = "test@1.2.3"
+
+
+def remove_handler_ignore(handler_name):
+    """Removes handlers of handlers ignored list."""
+    _IGNORED_LOGGERS.discard(handler_name)
+
+
+class TestException(exceptions.UserError):
+    pass
+
+
+class InMemoryTransport(HttpTransport):
+    """A :class:`sentry_sdk.Hub.transport` subclass which simply stores events in a list.
```

Comment:
Excuse the bureaucracy here, but apparently this has to be truncated from 89 to 88 lines according to new [ruff](https://github.com/OCA/server-tools/actions/runs/8970280465/job/24633463815) rules.

Reply:
Rebased and fixed.